src/bptl/work_units/kownsl/tasks.py

```diff
@@ -123,3 +144,194 @@ def get_approval_status(task: BaseTask) -> dict:
+    * ``kownslUsers``: list of usernames that have been configured in the review request configuration.
+
+    **Sets the process variables**
+
+    * ``remindThese``: a JSON-object containing a list of usernames who need reminding:
+
+      .. code-block:: json
+
+            [
+                "user1",
+                "user2",
+            ]
+    """
+
+    # Get the review request with id as given in variables
+    review_request = get_review_request(task)
+
+    review_request_id = review_request["id"]
+
+    # Get review request type to set operation_id
+    review_type = review_request["review_type"]
+    if review_type == "approval":
+        operation_id = "reviewrequest_approvals"
+    else:
+        operation_id = "reviewrequest_advices"
+
+    client = get_client(task)
+    url = get_operation_url(
+        client.schema,
+        operation_id,
+        base_url=client.base_url,
+        uuid=review_request_id,
+    )
+
+    # Get approvals/advices belong to review request
+    reviews = client.request(url, operation_id)
+
+    # Build a list of users that have responded
+    already_responded = []
+    for review in reviews:
+        user = review["author"]
+        already_responded.append(user)
+
+    # Check who should respond
+    variables = task.get_variables()
+    needs_to_respond = check_variable(variables, "kownslUsers")
+
+    # Finally figure out who hasn't responded yet
+    not_responded = [
+        username for username in needs_to_respond if username not in already_responded
+    ]
+    return {
+        "remindThese": not_responded,
+    }
+
+
+@register
+def get_review_request_reminder_date(task: BaseTask) -> dict:
+    """
+    Get the reminder for the set of reviewers who are requested.
+    The returned value is the deadline minus one day.
+
+    In the task binding, the service with alias ``kownsl`` must be connected, so that
+    this task knows which endpoints to contact.
+
+    **Required process variables**
+
+    * ``kownslReviewRequestId``: the identifier of the Kownsl review request.
+    * ``zaakUrl``: URL reference to the zaak used for the review itself.
+    * ``kownslUsers``: list of usernames that have been configured in the review request configuration.
+
+    **Sets the process variables**
+
+    * ``reminderDate``: a string containing the reminder date: "2020-02-29".
+    * ``deadline``: a string containing the deadline date: "2020-03-01".
+    """
+    # Get kownslUsers
+    variables = task.get_variables()
+    kownsl_users = check_variable(variables, "kownslUsers")
+
+    # Get the review request with id as given in variables
+    review_request = get_review_request(task)
+    user_deadlines = review_request["user_deadlines"]
+
+    # Get deadline belonging to that specific set of kownslUsers
+    deadline_str = user_deadlines.get(kownsl_users[0])
+    deadline = datetime.datetime.strptime(deadline_str, "%Y-%m-%d").date()
```

Comment:
```suggestion
    deadline_str = user_deadlines[kownsl_users[0]]
    deadline = datetime.datetime.strptime(deadline_str, "%Y-%m-%d").date()
```

Because:

```py
>>> import datetime
>>> deadline_str = None
>>> datetime.datetime.strptime(deadline_str, "%Y-%m-%d").date()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: strptime() argument 1 must be str, not None
```